src/pages/signup.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```

Comment:
this file should be called sign-up.page.js

Reply:
this page already exists and this is probably just excess now that can be deleted after